docs/CreateEKSClusterForScalarDLAuditor.md

```diff
@@ -0,0 +1,204 @@
+# Create an EKS cluster for ScalarDL Ledger and ScalarDL Auditor
+
+This document explains how to create an EKS cluster for the ScalarDL Ledger and ScalarDL Auditor deployment. For details on how to deploy ScalarDL Ledger and ScalarDL Auditor on the EKS cluster, please see [Deploy ScalarDL Ledger and ScalarDL Auditor on Amazon EKS (Amazon Elastic Kubernetes Service)](./ManualDeploymentGuideScalarDLAuditorOnEKS.md).
+
+## Steps
+
+You must create an EKS cluster based on the following requirements, recommendations, and your project's requirements. For details on how to create the EKS cluster, please see the [AWS official document](https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html).
+
+## Requirements
+
+There are some requirements for the deployment of ScalarDL Ledger and ScalarDL Auditor. Please configure your EKS cluster based on the following requirement and your project's requirements.
+
+* You must create the EKS cluster with Kubernetes version 1.21 or higher for the ScalarDL Ledger and ScalarDL Auditor deployment.
+* You cannot deploy your application pods on the same EKS cluster as ScalarDL Ledger and ScalarDL Auditor deployment to make Byzantine fault detection work properly.
+* You must create two EKS clusters. One is for ScalarDL Ledger another one is for ScalarDL Auditor.
+* You must configure a VPC as follows.
```

Comment:
In the ScalarDL Auditor mode document, users have to connect between two VPCs to keep administrative domains separate.